server/base/src/main/java/org/apache/accumulo/server/tablets/TabletTime.java

```diff
@@ -21,22 +21,20 @@
 
 import org.apache.accumulo.core.client.admin.TimeType;
 import org.apache.accumulo.core.data.Mutation;
+import org.apache.accumulo.core.metadata.schema.MetadataTime;
 import org.apache.accumulo.server.data.ServerMutation;
 import org.apache.accumulo.server.util.time.RelativeTime;
 
 public abstract class TabletTime {
-  public static final char LOGICAL_TIME_ID = 'L';
-  public static final char MILLIS_TIME_ID = 'M';
+  public static final char LOGICAL_TIME_ID = MetadataTime.getCode(TimeType.LOGICAL);
```

Comment:
Not completely sure, but I was thinking inlining this may be good.

Reply:
So wherever LOGICAL_TIME_ID is used instead use MetadataTime.getCode(TimeType.LOGICAL).  I started that way but then thought it might be cleaner to keep the constants. BUT...I also think it may be better to remove this Timetype from TabletTime.  It is really just a time calculator.  There is no need for the code to ask TabletTime for the Timetype.  The more I think about it the more I definitely think it should be removed.